ui/src/app/views/profile-page/component.css

```diff
@@ -140,3 +140,17 @@ hr {
 .error {
   justify-content: space-between;
 }
+
+.field-wrapper-aligned {
+  display: flex;
+  align-items: flex-start;
+}
+
+.error-margin {
+  margin-bottom: 16px;
```

Comment:
I would do a `margin-bottom: calc(total-margin - error-height)` to be a little more self documenting on what this is doing